jni/source/com_couchbase_lite_store_ForestDBStore.cpp

```diff
@@ -0,0 +1,100 @@
+#include <stdio.h>
+
+#include "com_couchbase_lite_store_ForestDBStore.h"
+
+
+#if !defined (CBL_KEY_CRYPTO_CC) \
```

Comment:
Instead of `CBL_KEY_CRYPTO_CC` and `CBL_KEY_CRYPTO_OPENSSL`, please use `_CRYPTO_CC` and `_CRYPTO_OPENSSL` for consistency with forestdb